OctopusDSC/DSCResources/cTentacleAgent/cTentacleAgent.psm1

```diff
@@ -51,11 +51,15 @@ function Get-MachineFromOctopusServer
 
         [Parameter()]
         [System.String]
-        $Instance
+        $Instance,
+
+        [Parameter()]
+        [System.String]
+        $SpaceId = "Spaces-1"
```

Comment:
I prefer to try and have only optional params on public methods, not the internal ones.
Otherwise, becomes a bit of a bug magnet.

Also marking all internal method params as mandatory helps avoid bugs too - we just haven't been too consistent with that.